src/plugins/timelist/Timelist.vue

```diff
@@ -369,32 +356,22 @@ export default {
     updateTimeStampAndListActivities(time) {
       this.timestamp = time;
 
-      this.listActivities();
-    },
-    filterActivities(activity, index) {
-      const hasFilterMatch = this.filterByName(activity.name);
-
-      if (hasFilterMatch === false || this.hideAll === true) {
-        return false;
-      }
+            this.listActivities();
+        },
+        filterActivities(activity, index) {
+            if (this.isEditing) {
+                return true;
+            }
 
-      if (this.showAll === true) {
-        return true;
-      }
+            const hasFilterMatch = this.filterByName(activity.name);
+            if (hasFilterMatch === false || this.hideAll === true) {
+                return false;
+            }
 
-      //current event or future start event or past end event
-      const isCurrent =
-        this.noCurrent === false &&
-        this.timestamp >= activity.start &&
-        this.timestamp <= activity.end;
-      const isPast =
-        this.timestamp > activity.end &&
-        (this.viewBounds.pastEnd === undefined ||
-          activity.end >= this.viewBounds.pastEnd(this.timestamp));
-      const isFuture =
-        this.timestamp < activity.start &&
-        (this.viewBounds.futureStart === undefined ||
-          activity.start <= this.viewBounds.futureStart(this.timestamp));
+            //current event or future start event or past end event
+            const isCurrent = (this.timestamp >= activity.start && this.timestamp <= activity.end);
```

Comment:
Complex boolean expressions should be split across multiple lines as it was previously.

Reply:
Running `npm run lint:fix` should fix this

Reply:
Prettier is adamant that this statement remains on 1 line; I tried breaking lines using parens but it deleted them since they are unnecessary chars.